docs/sphinx_qtile.py

```diff
@@ -195,12 +236,21 @@ def make_rst(self):
 def generate_keybinding_images():
     this_dir = os.path.dirname(__file__)
     base_dir = os.path.abspath(os.path.join(this_dir, ".."))
-    call(['make', '-C', base_dir, 'run-ffibuild'])
-    call(['make', '-C', this_dir, 'genkeyimg'])
+    run(['make', '-C', base_dir, 'run-ffibuild'])
+    run(['make', '-C', this_dir, 'genkeyimg'])
+
+
+def generate_widget_screenshots():
+    this_dir = os.path.dirname(__file__)
+    try:
+        run(['make', '-C', this_dir, 'genwidgetscreenshots'], check=True)
+    except CalledProcessError:
```

Comment:
If the screenshots fail to build, the docs will also fail.

Reply:
No that should be fine. Does anybody receive any notification, or would this become apparent as part of the CI?

Reply:
No I think that's fine. Will we get the red in CI on master, or any other notification?